### PR TITLE
OCD-1523: log output to file and stdout

### PR DESCRIPTION
**Pull request by dlucas1**

### CODE REVIEW

**Review by kekey1 — APPROVED** [review]

File: updateCertificationStatusApp.sh [hunk 1]
```diff
@@ -5,4 +5,20 @@
 # 15 5 * * * cd /tempApps && ./updateCertificationStatusApp.sh
 # This will run it at 0515 UTC, which (depending on DST) is 0015 EST
 
-java -jar certStatusApp-jar-with-dependencies.jar
\ No newline at end of file
+# create timestamp and filename
+TIMESTAMP=$(date "+%Y.%m.%d-%H.%M.%S")
+log=logs/log.certStatusApp.$TIMESTAMP.txt
```
Comment (andlar) [inline updateCertificationStatusApp.sh:10]: Does the `logs` directory exist? If not, this will fail

File: updateCertificationStatusApp.sh [hunk 2]
```diff
@@ -5,4 +5,20 @@
...
+
+# deal with spaces in filenames by saving off the default file separator (including spaces)
+# and using a different one for this application
```
Comment (andlar) [inline updateCertificationStatusApp.sh]: Why do this? Do you have spaces in the filename?

File: updateCertificationStatusApp.sh (hunk 1 above)
Comment (dlucas1) [reply updateCertificationStatusApp.sh:10]: It does exist on my local machine and on the Jenkins machine where this is deployed. However, for some reason git isn't noticing the added folder and reflecting it when I run "git status" even after doing a "git fetch origin".

File: updateCertificationStatusApp.sh (hunk 2 above)
Comment (dlucas1) [reply updateCertificationStatusApp.sh]: This was borrowed from the other shell files just to be safe. I think the application would save the log file just fine without this.

File: updateCertificationStatusApp.sh (hunk 1 above)
Comment (andlar) [reply updateCertificationStatusApp.sh:10]: As long as the directory is there on the machine that's going to run it that's fine. See if you can figure out why git doesn't track the directory though; it's a good thing to know

File: updateCertificationStatusApp.sh (hunk 2 above)
Comment (andlar) [reply updateCertificationStatusApp.sh]: "Just to be safe" is cause of so much code debt and cargo cult programming. If you _know_ why it's there, and choose to use it, that's fine, but if you don't, or if you're not sure, then it just means it's left there to confuse someone later on

**Review by andlar — APPROVED** [review]
Meant to approve this earlier but forgot

**Pull request merged**